algebra/src/sparse_linear_algebra.rs

```diff
@@ -1,15 +1,13 @@
-use algebra::Field;
+use crate::Field;
```

Comment:
Let's open an issue about finding a way to uniform the representation of the matrices. Once this is done, ideally, I see this file more part of the r1cs-core crate

Reply:
I have updated Marlin to use the same sparse matrix representation as the one used in the r1cs-core crate.